CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 ### unreleased
 
+### 0.2.0 (2020-06-02)
+
+* Multi-component support (thanks @mpickering)
+* Support for GHC 8.10 (thanks @sheaf)
```

Comment:
this was contributed by @chshersh iirc

Reply:
The first PR towards 8.10 support was https://github.com/digital-asset/ghcide/pull/360. I’ve added @chshersh as well now.